modules/cloudaccess/wf_peering_acceptor_definition.json

```diff
@@ -0,0 +1,16 @@
+{
+    "actions": [
+        "Microsoft.Authorization/roleAssignments/read",
+        "Microsoft.Authorization/roleDefinitions/read",
+        "Microsoft.Authorization/*/read",
+        "Microsoft.Insights/alertRules/*",
+        "Microsoft.Network/*",
```

Comment:
narrowing this needs to be high up our list of TODOs I suspect, given the sort of subscriptions that one would typically want to accept peering into.